spacy/ml/models/coref.py

```diff
@@ -56,7 +56,9 @@ def convert_coref_clusterer_inputs(model: Model, X: List[Floats2d], is_train: bo
     def backprop(args: ArgsKwargs) -> List[Floats2d]:
         # convert to xp and wrap in list
         gradients = torch2xp(args.args[0])
-        assert isinstance(gradients, Floats2d)
+        #TODO why did this change? This was fine before merging master.
+        # It's still 2d but this assert fails.
+        #assert isinstance(gradients, Floats2d)
```

Comment:
Merging in `master` broke this `assert` for some reason, I'm not sure why. I need to look at this more.

Reply:
Probably because of a change in `8.1.0.dev0`?

Reply:
I considered the possibility of Thinc changes, but I haven't updated my dev environment, so that's not the case. I'm using Thinc 8.0.16.

Reply:
This is the only thing here I still don't totally understand, but it doesn't need to hold up this PR. (I'm also not sure why the assert was here, @kadarakos was there an error case or something this was checking for?

Reply:
I don't think I've put the assert there I also have no idea.

Reply:
OK, I'll just remove it.